lib/core/src/Cardano/Wallet/Primitive/Model.hs

```diff
@@ -531,7 +531,7 @@ prefilterBlock b u0 = runState $ do
 
                 (Nothing, Outgoing) -> -- Byron
                     let
-                        totalOut = sumCoins (txOutCoin <$> outputs tx)
+                        totalOut = F.fold (txOutCoin <$> outputs tx)
```

Comment:
> I appreciate the clarity of the name sumCoins over F.fold, but I won't per-se object with this change either.

I think my (not very strongly felt) objection to this stems from the fact that `sumCoins` seems specialized, but in reality it folds over the coins in a very similar way to how `F.fold` adds with the `Semigroup` and `Monoid` instances, so it _seems_ a bit superfluous.

But my objection is not a strong one! I would prefer the name `sum` though, if we were to retain it, since the current API of `Coin` is intended to be imported in qualified fashion.